MekHQ/src/mekhq/gui/view/AtBContractViewPanel.java

```diff
@@ -495,8 +491,17 @@ private void fillStats() {
         } else {
             lblSalvagePct.setText(resourceMap.getString("lblSalvagePct.text"));   
             int maxSalvagePct = contract.getSalvagePct();
-            int currentSalvagePct = (int)(100*((double)contract.getSalvagedByUnit())/(contract.getSalvagedByUnit()+contract.getSalvagedByEmployer()));
-            txtSalvagePct.setText(currentSalvagePct + "% (max " + maxSalvagePct + "%)");       
+
+            int currentSalvagePct = 0;
+            if (contract.getSalvagedByUnit().plus(contract.getSalvagedByEmployer()).isPositive()) {
+                currentSalvagePct = contract.getSalvagedByUnit()
+                        .dividedBy(contract.getSalvagedByUnit().plus(contract.getSalvagedByEmployer()).getAmount().doubleValue())
```

Comment:
would benefit from a dividedBy(Money) overload

Reply:
I thought the same, but interestingly that doesn't exist in joda.money. Not sure why to be honest. I could add it and unwrap the amount inside to clean the API for consumers.